spec/models/hackney/note.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe Hackney::Note do
+
+  def note_response_body
```

Comment:
Why did you split this body into a method? I can see that it's only used as a default argument to `stub_get_notes`, which is never overridden, and that `stub_post_a_note` does not have it's body split in a separate method.

Reply:
👍